README.md

```diff
@@ -120,43 +151,16 @@ installation command
 ```
 
 ## Running
-After installing requirements and activating your environment, there is an easy
-to use `prompt.py` file for you to run the node as well as some basic interactivity
 
-```
-python prompt.py
-NEO cli. Type 'help' to get started
-
-neo> show state
-Progress: 1054913 / 1237188
+After installing requirements and activating your environment, 
+there is an easy to use `prompt.py` file for you to run the node 
+as well as some basic interactivity:
 
-neo>
-```
+![opening prompt.py](http://i.imgur.com/jQklFSB.png)
```

Comment:
Same issue as above, would like to keep the main readme as simple as possible, and put more information in a docs build.